adal/authentication_context.py

```diff
@@ -193,6 +193,10 @@ def acquire_token_with_authorization_code(self, authorization_code,
         :param str client_secret: (only for confidential clients)The OAuth
             client secret of the calling application. This parameter if not set,
             defaults to None
+        :param str code_verifier: (optional)The code verifier that was used to
+           obtain authorization code if PKCE was used in the authorization

```

Comment:
Sphinx can probably still accept the current docstring as-is, but culture-wise, Python community pays great attention to the [indentation](https://stackoverflow.com/a/50455731/728675). It doesn't feel "right" when an indentation does not match 4-spaces, and/or same paragraph does not use same indentation.